index.html

```diff
@@ -56,9 +65,30 @@ <h2 class="tonic">Tonic</h2>
 					<li class="btn3">javaScript</li>
 				</ul>
 				<button class="btn4" type="submit">See Project</button>
+				</div>
 			</div>
 			<div class="works2">
-				<img class="snap2" src="images/Snap4.png" alt="">
+				<img class="snap2" src="images/Snapshoot2.png" alt="">
+				<img class="snap-desk2" src="images/Snap2.png" alt="">
+				<div class="desk-work2">
+				<h2 class="tonic2">Multi-Post Stories</h2>
+				<div class="grid-container2">
+				  <div class="grid-item12"><p>FACEBOOK</p></div>
+				  <div class="grid-item22"><p>.</p></div>
+				  <div class="grid-item32"><p>Full Stack Dev</p></div>
+				  <div class="grid-item42"><p>.</p></div>
+				  <div class="grid-item52"><p>2015</p></div>
+				</div>
+				<p class="day2">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+				<ul class="langs2">
+					<li class="btn12">html</li>
+					<li class="btn52">Ruby on rails</li>
+					<li class="btn22">css</li>
+					<li class="btn32">javaScript</li>
```

Comment:
- [ ] Please decrease the gap between the tags and make sure it matches the Figma design.

### Your design
![image](https://user-images.githubusercontent.com/83803414/186701269-57e7c6b9-74d3-451d-9454-4cd92d0a9bda.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/186701430-8d07d7ed-8bdd-47e8-ba0e-5d0235a36266.png)